src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Move.cs

```diff
@@ -16,11 +20,41 @@ public void Directory_Move_To_Same_Directory()
         }
 
         [Fact]
+        [PlatformSpecific(TestPlatforms.OSX)]
```

Comment:
The test will pass however due to the filter it can pass even when it is actually broken.

Reply:
If the test will pass on all other platforms, I suggest just letting it run on all platforms.

Otherwise, since it's not obvious why it's being limited to just macOS, please add a comment.